app/parse.py

```diff
@@ -8,5 +14,28 @@ class Course:
     duration: str
 
 
+def parse_single_course(soup: Tag) -> Course:
+    return (Course(
+        name=soup.select_one("a.ProfessionCard_title__fEWio > h3").text,
+        short_description=soup.select_one(
+            ".ProfessionCard_professionTags__2iarD + "
+            "p.typography_landingTextMain__Rc8BD"
+        ).text,
+        duration=soup.select_one(
+            ".ProfessionCardTags_regularTag__dqOGj:last-child span"
+        ).text
+    ))
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    response = requests.get(BASE_URL).content
```

Comment:
Ensure that `BASE_URL` points directly to the page containing the courses. If it redirects or requires additional parameters, the request might not return the expected content.